src/components/SocialShareButtons.vue

```diff
@@ -17,7 +17,7 @@
         class="social-button pinterest"
         target="_blank"
         @click="onSocialMediaLinkClick('Pinterest')"
-        :href="`https://www.pinterest.com/pin/create/bookmarklet/?media=${imageURL}&description=${shareText}`">
+        :href="`https://www.pinterest.com/pin/create/bookmarklet/?media=${imageURL}&is_video=false&description=${shareText}`">
```

Comment:
These changes are causing unit testing to fail. Please address this, here's the error info:
```
Summary of all failing tests
 FAIL  test/unit/specs/components/social-share-buttons.spec.js (5.23s)
  ● SocialShareButtons › should render all buttons with correct query params

    expect(string).toContain(value)
    
    Expected string:
      "<a target=\"_blank\" href=\"https://www.pinterest.com/pin/create/bookmarklet/?media=http://image.url&amp;is_video=false&amp;description=share text\" class=\"social-button pinterest\"></a>"
    To contain value:
      "?media=http://image.url&amp;description=share text"

      35 |     expect(wrapper.find('.social-button.facebook').html()).toContain('?u=http://share.url&amp;t==share');
      36 |     expect(wrapper.find('.social-button.twitter').html()).toContain('?text=share text');
    > 37 |     expect(wrapper.find('.social-button.pinterest').html()).toContain('?media=http://image.url&amp;description=share text');
      38 |   });
      39 | 
      40 |   it('dispatches social media share event when facebook link is clicked', () => {
      
      at Object.<anonymous> (test/unit/specs/components/social-share-buttons.spec.js:37:61)
```